jwst/datamodels/tests/test_models.py

```diff
@@ -239,6 +239,16 @@ def test_dtype_match():
         dm.data = np.array([[1, 2, 3]], np.float32)
 
 
+def test_default_value():
+    with ImageModel((100, 100)) as im:
```

Comment:
Looks like this test makes sure that attributes in `meta` that have `anyOf` in their schemas now produce correct default values, i.e. a scalar.  Before they returned an `ObjectNode`, as datamodels didn't see these as endpoints in the tree, but as yet another node.

Reply:
It would be nice to indicate that this test also tests attributes that have `anyOf` schemas.  Maybe rename it to
```
def test_default_value_for_anyof_schemas():
```

Reply:
Or put a docstring in the test function to indicate it has to do with `anyOf` schemas.